apps/src/calc/calc.js

```diff
@@ -143,16 +154,9 @@ Calc.init = function(config) {
         config.forceInsertTopBlock);
     }
 
-    appState.targetExpressions = generateExpressionsFromBlockXml(solutionBlocks);
+    appState.targetSet = generateEquationSetFromBlockXml(solutionBlocks);
```

Comment:
Maybe s/`targetSet`/`targetEquationSet` and s/`userSet`/`userEquationSet` to clarify type